File: javascript/2164.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var n = parseInt(input);

const fibonacci = (Math.pow((1 + Math.sqrt(5)) / 2, n) - Math.pow((1 - Math.sqrt(5)) / 2.0, n)) / Math.sqrt(5);
console.log(fibonacci.toFixed(1));

File: C#/2164.cs
using System; 

class URI {

    static void Main(string[] args) { 
            
        int n;
        double fibonacci;
        
        n = int.Parse(Console.ReadLine());
    
        fibonacci = (Math.Pow((1 + Math.Sqrt(5)) / 2, n) - Math.Pow((1 - Math.Sqrt(5)) / 2.0, n)) / Math.Sqrt(5);
        Console.WriteLine($"{fibonacci:#.0}");
    }
}